spec/models/mainstream_browse_page_spec.rb

```diff
@@ -12,97 +10,99 @@
       "links" => {
       },
     }
-    @content_item = ContentItem.new(@api_data)
-    @page = MainstreamBrowsePage.new(@content_item)
   end
+  let(:content_item) { ContentItem.new(api_data) }
+  let(:page) { described_class.new(content_item) }
 
   describe "basic properties" do
     it "returns the browse page content_id" do
-      assert_equal "f818b54c-01c7-43e9-8165-cec12e4053ff", @page.content_id
+      expect(page.content_id).to eq("f818b54c-01c7-43e9-8165-cec12e4053ff")
     end
 
     it "returns the browse page base_path" do
-      assert_equal "/browse/benefits/child", @page.base_path
+      expect(page.base_path).to eq("/browse/benefits/child")
     end
 
     it "returns the browse page title" do
-      assert_equal "Child Benefit", @page.title
+      expect(page.title).to eq("Child Benefit")
     end
 
     it "returns the browse page description" do
-      assert_equal "Information about eligibility, claiming and when Child Benefit stops", @page.description
+      expect(page.description).to eq("Information about eligibility, claiming and when Child Benefit stops")
     end
 
     describe "#second_level_pages_curated?" do
       it "is true when second_level_ordering == curated" do
-        @api_data["details"]["second_level_ordering"] = "curated"
-        assert @page.second_level_pages_curated?
+        api_data["details"]["second_level_ordering"] = "curated"
+        expect(page).to be_second_level_pages_curated
```

Comment:
`expect(page).to be_second_level_pages_curated` works, but I wonder if it's clearer to say:

`expect(page.second_level_pages_curated?).to be(true)`, not least because we use this pattern to assert false just below